terraform/modules/concourse_web/iam.tf

```diff
@@ -9,13 +9,13 @@ resource "aws_iam_instance_profile" "web" {
   role = aws_iam_role.web.id
 }
 
-resource "aws_iam_role_policy_attachment" "ssm" {
-  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonEC2RoleforSSM"
+resource "aws_iam_role_policy_attachment" "cloudwatch_logging" {
```

Comment:
I really prefer us not trying to build up our own ARNs :)

Can we create data sources for all the policies we want to attach, then grab the ARN from those please?

Reply:
I don't know what you mean here? Also feels a little out of scope for what I'm trying to achieve?